spec/lib/guard/notifier_spec.rb

```diff
@@ -41,4 +41,42 @@
       end
     end
   end
+
+  describe "backward compatibility" do
+    let(:notifier) { instance_double("Notiffany::Notifier") }
+
+    before do
+      allow(Notiffany::Notifier).to receive(:new).and_return(notifier)
+    end
+
+    after do
+      Guard::Notifier.instance_variable_set(:@notifier, nil)
```

Comment:
This is generic also (can be extracted out to the top) - it should be done after every test in this file anyway.